src/data/core/vnotex.json

```diff
@@ -16,8 +16,8 @@
             "NewNote" : "Ctrl+Alt+N",
             "NewFolder" : "Ctrl+Alt+S",
             "CloseTab" : "Ctrl+G, X",
-            "CloseOtherTabs" : "",
-            "CloseTabsToTheRight" : "",
+            "CloseOtherTabs" : "Ctrl+G, F",
```

Comment:
Do not add default values for these two shortcuts.

Reply:
done